### ISSUE TITLE
docs(aio) incorrect information in ngmodule-faq.md

### ISSUE THREAD

**Issue by xuemind** [angular/angular#29468]
# 📚 Docs or angular.io bug report

### Description

in this doc, it told me as following.
> Lazy-loaded modules and their components can inject AppModule services; they can't inject AppComponent services.

But, according to my test, the components of lazy-loaded modules can inject the AppComponent services.

## 🔬 Minimal Reproduction
I modified the example lazy-loaded module from Angular guide for demonstration.
https://stackblitz.com/edit/lazy-loaded-injector

### What's the affected URL?**
 

[Should I add application-wide providers to the root AppModule or the root AppComponent](https://angular.io/guide/ngmodule-faq#should-i-add-application-wide-providers-to-the-root-appmodule-or-the-root-appcomponent)

https://github.com/angular/angular/blob/master/aio/content/guide/ngmodule-faq.md

### Reproduction Steps**

1. create a `DummyService` with no `providedIn` and some logs info.
2. make sure it is not in `providers` of `AppModule` and lazy-loaded module, e.g. `OrdersModule`. 
3. Put `DummyService` in `providers` list of the root component, `AppComponent` . 
4. The the DummyService also can be injected into `OrdersModule` belong to lazy-loaded module `OrdersModule`
5. check the console log
```
🐌🐌🐌 Dummy (1553284472555) created without parent
🍅🍅🍅 Dummy (1553284472555) injected into AppComponent
🍅🍅🍅 Dummy (1553284472555) injected into OrderListComponent
```
So, the instance of DummyService was created by the Injector of root component `AppComponent` and also re-used by `OrderListComponent` which belongs to a lazy-loaded module. The behavior is different with description in the doc.

## 🔥 Exception or Error
<pre><code>

</code></pre>

## 🌍  Your Environment
Angular CLI: 7.3.5
Node: 10.15.1
OS: win32 x64
Angular: 7.2.9
... animations, common, compiler, compiler-cli, core, forms
... http, platform-browser, platform-browser-dynamic
... platform-server, router, upgrade

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.13.5
@angular-devkit/build-angular     0.13.5
@angular-devkit/build-optimizer   0.13.5
@angular-devkit/build-webpack     0.13.5
@angular-devkit/core              7.3.5
@angular-devkit/schematics        7.3.5
@angular/cli                      7.3.5
@ngtools/webpack                  7.3.5
@schematics/angular               7.3.5
@schematics/update                0.13.5
rxjs                              6.4.0
typescript                        3.1.6
webpack                           4.29.0

**Comment by xuemind** [comment]
I updated the report with sample code here: https://stackblitz.com/edit/lazy-loaded-injector

**Comment by kapunahelewong** [comment]
Hi @xuemind, thank you for this issue. I went through your example and to try it for myself, I created an even simpler one to test it but my example behaves as expected. I compared it, your example, and your reproduction steps, but I'm not seeing the difference between yours and mine. Does mine help you? [https://stackblitz.com/edit/angular-issue-29468](https://stackblitz.com/edit/angular-issue-29468)
